### PR TITLE
Wrap result display

### PR DESCRIPTION
**Pull request by harithh07**

### CODE REVIEW

File: src/main/resources/view/ResultDisplay.fxml [hunk 1]
```diff
@@ -5,5 +5,5 @@
 
 <StackPane fx:id="placeHolder" xmlns="http://javafx.com/javafx/17"
     xmlns:fx="http://javafx.com/fxml/1">
-  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display"/>
+  <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display" wrapText="true"/>
```
Comment (kvernnn) [inline src/main/resources/view/ResultDisplay.fxml:8]: Looks good, should the command box have a wrap text too?

**Review by kvernnn — COMMENTED** [review]
LGTM, suggestion to consider.

File: src/main/resources/view/ResultDisplay.fxml (hunk 1 above)
Comment (harithh07) [reply src/main/resources/view/ResultDisplay.fxml:8]: Good idea, but I believe the text wrapping will look unappealing as the size of the command box would change.

Comment (kvernnn) [reply src/main/resources/view/ResultDisplay.fxml:8]: Okay.

**Pull request merged**